09-mapping.Rmd

```diff
@@ -542,6 +573,8 @@ The following command creates the animation illustrated in Figure \@ref(fig:urba
 tmap_animation(urb_anim, filename = "urb_anim.gif", delay = 25)
 ```
 
+<!-- Though the second animation is great, I wonder if it is necessary here, especially if the underlying code is not explained in more detail. So this could be also a great challenge for the exercise section?
+-->
 Another illustration of the power of animated maps is provided in Figure \@ref(fig:animus).
```

Comment:
Though the second animation is great, I wonder if it is necessary here, especially if the underlying code is not explained in more detail and considering the length the viz chapter already has (which even more content to come). So this could be also a great challenge for the exercise section?

Reply:
I disagree: I think we should show people what is possible, encourage them to reproduce by linking to the code and inspire them to change the world ;)

Reply:
I disagree: I think we should show people what is possible, encourage them to reproduce by linking to the code and inspire them to change the world ;)